src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,78 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private String dataFromFile;
+    private String dataToFile;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        getDataFromFile(fromFileName);
+        convertInputDataInOutputData();
+        setDataToFile(toFileName);
+    }
+
+    private void getDataFromFile(String fromFileName) {
+        StringBuilder stringBuilder = new StringBuilder();
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            int value = reader.read();
+            while (value != -1) {
+                stringBuilder.append((char) value);
+                value = reader.read();
+            }
+        } catch (FileNotFoundException e) {
+            throw new RuntimeException("No such file" + e);
+        } catch (IOException e) {
+            throw new RuntimeException("Error in file reading" + e);
+        }
+        dataFromFile = stringBuilder.toString();
+    }
+
+    private void convertInputDataInOutputData() {
+        int supply = 0;
+        int buy = 0;
+        String[] dataFromFileInArr = dataFromFile.split("\r\n");
+        String switchCase;
+        for (String s : dataFromFileInArr) {
+            switchCase = s.substring(0, s.indexOf(','));
```

Comment:
Why do we use 0 and "," numbers here? I don't know about it
Let's create a constants